src/main/java/gui/sub_controllers/ZoomController.java

```diff
@@ -71,15 +72,15 @@ public void traverseGraphClicked(int centreNode, int radius) {
         int column = graph.getNode(centreNode).getColumn();
         drawer.changeZoom(column, radius);
         drawer.highlight(centreNode);
-        panningController.setScrollbarSize(drawer.getColumnWidth(column));
     }
 
     /**
      * Displays the centre node and radius of the current view.
      */
     public void displayInfo() {
-        nodeTextField.setText(drawer.getRealCentreNode().getId() + "");
-        radiusTextField.setText((int) Math.ceil(drawer.getRadius()) + "");
+        // commented out becaus it glitches out the graph drawer, drawing graphs on eachother.
```

Comment:
Are you sure? If that's so remove the code enturely like Lex says, otherwise, leave it in.